Exercises/promise-async-await/promise-async-await.js

```diff
@@ -1,7 +1,54 @@
+/*eslint-disable no-console*/
+//Tasks:
+//1. Make function which will return promise which will resolve in passed to function amount of milliseconds.
+const util = require('util');
+
+function callFunction(func, ms) {
+  return new Promise(resolve => {
+    console.log('Start: ');
+    setTimeout(() => {
+      console.log('End: ');
+      try {
+        resolve(func);
+      } catch (error) {
+        console.log(error);
+      }
+    }, ms);
+  });
+}
 const fakeAjax = async () => {
-  if (Math.random() > 0.5) {
+  const random = Math.random();
+  console.log(random);
+  if (random > 0.5) {
     return { status: 200 };
   }
 
   throw new Error('Server error!');
 };
+//2. Using `fakeAjax` do 3 parallel requests.
+//3. Catch errors but only for failed executions.
+const firstPromise = util.promisify(fakeAjax);
+const secondPromise = util.promisify(fakeAjax);
+const thirdPromise = util.promisify(fakeAjax);
+
+async function finalResults() {
+  await Promise.all([firstPromise(), secondPromise(), thirdPromise()]).then(response => {
```

Comment:
Currently error in any single promise will reject whole Promise.all execution. You need to catch errors where it's needed and still to be able to finish the rest promises